operations/mimir-mixin/dashboards/dashboard-queries.libsonnet

```diff
@@ -0,0 +1,9 @@
+{
+  // This object contains common queries used in the Mimir dashboards.
+  // These queries are NOT intended to be configurable or overriddeable via jsonnet,
+  // but they're defined in a common place just to share them between different dashboards.
```

Comment:
If these are really NOT intended to be configurable or overriddeable, I'd enforce that in their usage. 

If you export the plain object here:

```jsonnet
{
  write_http_routes_regex: 'api_(v1|prom)_push|otlp_v1_metrics',
  read_http_routes_regex: '(prometheus|api_prom)_api_v1_.+',
}
```

And then use it as:

```jsonnet
local queries = import 'dashboard-queries.libsonnet';

//...
{bar: foo(queries.libsonnet)}
//...
```

It will be impossible for a user to override them.

Reply:
It's not intended, but at this stage I wouldn't even go that far to make it really impossible. I just didn't want to make it part of the `_config`.

Reply:
We can revisit this any time tho.

Reply:
Sure, it wasn't a blocker in any way, just sharing some ideas.